doc/api/cli.md

```diff
@@ -1028,6 +1028,12 @@ Valid values are `"commonjs"` and `"module"`. The default is `"commonjs"`.
 
 The REPL does not support this option.
 
+Additional details:
+
+* The default input type can be altered with
+  [`--experimental-default-type=module`][].
```

Comment:
nit: I think it makes more sense to link to the flag, rather than a specific value for this flag (in case e.g. more values were added in the future)
```suggestion
  [`--experimental-default-type`][].
```

Reply:
`--experimental-default-type=module` also links to main type section. I have added a new link to `--experimental-default-type=type`